release/gcb/cloud_builder.sh

```diff
@@ -45,6 +45,10 @@ export GOPATH
 echo gopath is "$GOPATH"
 # this is needed for istioctl and other parts of build to get the version info
 export ISTIO_VERSION="${CB_VERSION}"
+export VERSION="${CB_VERSION}"
```

Comment:
VERSION is set specifically by this cloud_builder.sh script.  I am not sure why `SITIO_VERSION` is set when `VERSION` is the variable used in the makefile.  I think there is an opportunity here for further simplification.  Whether we tackle that in this already difficult to review PR or not is up to the reviewers :)